.eslintrc.js

```diff
@@ -4,7 +4,12 @@
 module.exports = {
     root: true,
     extends: ["plugin:@eslint-community/mysticatea/es2020"],
+    parserOptions: {
+        project: "./tsconfig.json",
```

Comment:
Nit: any reason not to use the nearest always? These should end up being the same result, just a little cleaner config IMO:

```suggestion
        project: true,
```